uninstall

```diff
@@ -272,7 +281,7 @@ paths = %w[bin etc include lib opt sbin share var
         .map { |p| HOMEBREW_PREFIX/p }
         .select(&:exist?)
         .map(&:to_s)
-if paths.any?
+if @is_mac && paths.any?
   args = paths + %w[-name .DS_Store]
```

Comment:
```suggestion
if paths.any?
  args = paths
  args += %w[-name .DS_Store] if @is_mac 
```

Reply:
I can see now that there's a bug in my original change, but shouldn't the logic in this section be:

```
if paths.any?

  # Delete .DS_Store on macOS only...
  if @is_mac
    args = paths + %w[-name .DS_Store]
    if options[:dry_run]
      args << "-print"
    else
      args << "-delete"
    end
    puts "Would delete:" if options[:dry_run]
    sudo "/usr/bin/find", *args
  end

  # ...but remove all empty subdirs on all OSes
  args = paths + %w[-depth -type d -empty]
  if options[:dry_run]
    args << "-print"
  else
    args += %w[-exec rmdir {} ;]
  end
  puts "Would remove directories:" if options[:dry_run]
  sudo "/usr/bin/find", *args

end
```

Reply:
Yup, rereading it again: I agree with your newly suggested version 👍